examples/test-cpu-arch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
```

Comment:
instead of adding this to the root of the `examples` folder, could we put it in `examples/support-bundle`?

Reply:
@laverya yes, that is a valid point. I've moved this to `examples/support-bundle` in my next change.